package.json

```diff
@@ -188,7 +188,7 @@
     "react-redux-promise-listener": "^1.0.0",
     "react-router-dom": "^5.1.2",
     "react-tabs": "^3.1.0",
-    "react-textarea-autosize": "^7.1.2",
+    "react-textarea-autosize": "^8.3.3",
```

Comment:
I haven't checked the new version properly, but it seems to be working without issues.

Let's hope this upgrade doesn't break something we're not aware of :)